config/Mekanism/generators.toml

```diff
@@ -82,7 +82,7 @@
 	[generators.fusion_reactor]
 		#The fraction of the heat dissipated from the case that is converted to Joules.
 		#Range: 0.0 ~ 1.0
-		thermocoupleEfficiency = 0
+		thermocoupleEfficiency = 0.007

```

Comment:
Would a value of 0.014 translate to roughly double the values in the PR description?

Reply:
... maybe?

Reply:
I'll test it here and let you know

Reply:
Short answer, no.

Long answer.. it's going to take a few minutes for the heat to stabilize to get a good read.

Reply:
Well, ok. I'm a dirty liar. It looked like it was going to grow faster than linearly after the initial adjustment, but it has stabilized off to pretty much exactly double the air cooled value with an efficiency of 0.014.

Reply:
😄 I would like the value to be `0.014` then, seems fair considering the injection rate I think. If I'm understanding correctly :P

Reply:
And up she goes